common/lib/xmodule/xmodule/js/spec/video/video_player_spec.js

```diff
@@ -388,148 +388,90 @@ function (VideoPlayer) {
                         spyOn(state.videoPlayer, 'stopTimer');
                         spyOn(state.videoPlayer, 'runTimer');
                         state.videoPlayer.seekTo(10);
-                    });
-
-                    waitsFor(function () {
-                        return state.videoPlayer.currentTime >= 10;
-                    }, 'currentTime is less than 10 seconds', WAIT_TIMEOUT);
-
-                    runs(function () {
-                        expect(state.videoPlayer.stopTimer)
-                            .toHaveBeenCalled();
-                        expect(state.videoPlayer.runTimer)
-                            .toHaveBeenCalled();
+                        jasmine.Clock.tick(300);
```

Comment:
Please add a comment about video player's usage of debounce and the importance of the 300 ms value.